### ISSUE TITLE
Test cognexa images regularly

### ISSUE THREAD

**Issue by blazekadam** [Cognexa/dockerfiles#14]
Either test the images nightly or test before pushing to the registry?

**Comment by FloopCZ** [comment]
What kind of tests do you propose? Some python stuff working, cxflow working, cxtream working, etc? Maybe we could run the tests of our projects?

It should not be necessary to test them before pushing, since if the tests fail, the image is not pushed anyway.

**Comment by blazekadam** [comment]
Well, one way to address it would be to include those images in our test matrix in **cxflow**, **cxflow-tensorflow** and **cxtream**. Not sure how to test if cuda works...

**Comment by petrbel** [comment]
Yeah, let's add more test units for OSS projects (cxflow, ..) that use these images directly instead of installing cxflow manually (at least nightly builds in master).
Cuda will be problem unless we somehow fool the OS to think it has a fake GPU...

All real project dockerfiles should be tested in a way they don't even compile in case of something went wrong. I suggest running the project tests as the last dockerfile command, however, @Teyras pointed out some problems such as increased image size (?), so this should be discussed.
